.env

```diff
@@ -0,0 +1 @@
+PORT = 3001
```

Comment:
- [ ] **Optional:** This file which is usually for app secrets is often ignored and not pushed to GitHub. Kindly note this for future projects.